ttyclock.c

```diff
@@ -98,7 +99,6 @@ init(void)
      if(ttyclock.option.utc) {
           ttyclock.tm = gmtime(&(ttyclock.lt));
      }
-     ttyclock.lt = time(NULL);
```

Comment:
do we really need to move the `.lt` initialization up there? it seems confusing because now it's not next to the (related) `.tm` setup.

Reply:
True. This is a residual change from an intermediate step.